### PR TITLE
Incorporate some pricing feedback

### PR DESCRIPTION
**Pull request by davidfischer**
- Make cards the same size (use decks) - also on learning hub
- Make "inquire" a link
- Have a "buy now" button
- Move some pricing notes around
- Ditch the questions below pricing

### CODE REVIEW

File: ethicalads-theme/templates/ea/advertisers.html [hunk 1]
```diff
@@ -28,10 +28,6 @@ <h1 class="display-3 font-weight-light" data-aos="fade-left">Reach millions of e
         <!-- Text -->
         <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">Get in front of a technical audience across an ecosystem of software-focused content</p>
 
-        <a href="#inbound-form" class="btn btn-primary lift" data-aos="fade-left" data-aos-delay="200">
-          <span>Start a campaign</span>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/advertisers.html:32]: Do we really want to remove the above the fold action?

File: ethicalads-theme/templates/ea/advertisers.html [hunk 2]
```diff
@@ -140,29 +136,6 @@ <h3>Hiring developers?</h3>
 {% include 'ea/includes/packages.html' %}
 
 
-<!-- Sign up promos -->
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/advertisers.html:143]: Do we want to remove this as well? Seems worth having.. Maybe put it under the "advertise with us" form? Or link to the FAQ or learning hub in some other fashion?

File: ethicalads-theme/templates/ea/includes/packages.html [hunk 3]
```diff
@@ -78,6 +79,12 @@ <h2 class="text-center font-weight-bold mb-4">Pricing</h2>
 
     </div> <!-- / .row -->
 
-    <p class="text-center small">For other region prices or CPC-based pricing, please inquire for details.</p>
+    <p class="text-center small"><sup>1</sup> Global buys cover all regions and have a small, customizable exclude list.</p>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/includes/packages.html:82]: This should still link to the FAQ, no?

**Review by ericholscher — COMMENTED** [review]
Hard to review this fully without a live preview of some kind, but the style changes look good. I'm less sold on the content changes, especially removing calls to action.

File: ethicalads-theme/templates/ea/advertisers.html (hunk 1 above)
Comment (davidfischer) [reply ethicalads-theme/templates/ea/advertisers.html:32]: NO. I removed the wrong one.

File: ethicalads-theme/templates/ea/includes/packages.html (hunk 3 above)
Comment (davidfischer) [reply ethicalads-theme/templates/ea/includes/packages.html:82]: I don't think it details any more than what's in the FAQ at least right now.

File: ethicalads-theme/templates/ea/advertisers.html (hunk 2 above)
Comment (davidfischer) [reply ethicalads-theme/templates/ea/advertisers.html:143]: This section isn't particularly useful. I'd rather funnel people into the form than list our email here. While the prospectus is useful, just about everything there is now on the website. I'm not sure this section adds anything.

**Review by ericholscher — APPROVED** [review]
Solid iteration -- definitely needs more love, but this is useful as is 👍

**Pull request merged**